### PR TITLE
GNU-style manual for org-trello

### PR DESCRIPTION
**Pull request by daniel-birket**
Mr. Dumont,

Please see this barely-begun template of a GNU-style manual for your org-trello package. Assuming that you approve of me investing the time to complete it, it will eventually produce an info-format manual for use within Emacs (like Emacs' own manual), plus HTML,  PDF and other format manuals, all from the same org-mode source. I'll pull the content from your source and doc-strings. See graphviz-dot-mode for a finished example (from .texi source).

If you're interested, I'll work on it, time permitting, for your eventual review.

### CODE REVIEW

**Comment by ardumont** [comment]
That sounds great.
Many thanks in advance for your time on this.

I don't know if you are aware of the online "manual" [1]
It sure is not perfect but that could be a start (the videos are a bit old but the idea is there ;)

That reminds me of some years back when i tried to do some literate programming (prior to 0.2.6 according to git log).

[1] http://org-trello.github.io/

Cheers,

File: texinfo/Makefile [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# tools
+TexInfoDir	:= /usr/local/Cellar/texinfo/6.5/bin
+
+# Source
+BaseName	:= org-trello
+OrgSrc		:= $(BaseName).org
+TexiSrc		:= $(BaseName).texi
+TexiInc		:= gpl.texi fdl.texi
```
Comment (ardumont) [inline texinfo/Makefile]: Looks like indendation is off?

**Comment by ardumont** [comment]
Thanks, looks like it's going the right way ;)
I have only a small remark about the indentation in the makefile but that's about it so far.

I have non-blocking questions about:
- how to compile this? That could be added in a README-doc.org (or something) to explain how it's done. That sure would be great ;)

- how to package this? The current packaging is done for el-get (~ recipe), melpa (~ recipe), marmalade (~ tarball created through makefile and push).

Cheers,

**Comment by ardumont** [comment]
Thanks for your work on this, and many thanks for clarifying the manual build instructions ;)

File: texinfo/org-trello.org [hunk 2]
```diff
@@ -498,21 +503,29 @@ list items.
 :END:
 This section describes other functions not listed above.
 
-** TODO How to Perform Tasks
+** TODO How to…
 :PROPERTIES:
 :ALT_TITLE: How to…
 :DESCRIPTION: How to Perform Tasks
 :END:
 This section is describes how to use commands to perform tasks with org-trello.
```
Comment (ardumont) [inline texinfo/org-trello.org]: is describing

Comment (ardumont) [reply texinfo/org-trello.org]: or `describes` as the section below does ;)

File: texinfo/org-trello.org [hunk 3]
```diff
@@ -603,16 +616,25 @@ remove specific existing org-trello keybindings:
...
+:END:
+
+** Automatic org-trello Files
+:PROPERTIES:
+:ALT_TITLE: Auto Files
+:DESCRIPTION: Automatic org-trello Files
+:END:
 
-* Contributing to org-trello
+* CHEK Development of org-trello
```
Comment (ardumont) [inline texinfo/org-trello.org]: CHECK?

I see that you use CHEK consistently everywhere (at least in this commit).
What does that convey?

**Comment by daniel-birket** [comment]
Ack. I will review aloud eventually when more draft done. Thank you.


________________________________

File: texinfo/org-trello.org [hunk 4]
```diff
@@ -636,16 +658,162 @@ and thanks to all those that opened [[https://github.com/org-trello/org-trello/i
...
+:ALT_TITLE: Mailing List
+:DESCRIPTION: Mailing List
+:END:
+
+This project has a mailing list at librelist.com for communication
+and community among its users and contributors. You may join the
+list by sending an email to:
+
+- subscribe: emacs.org.trello [AT] librelist.com
+- unsubscribe: emacs.org.trello-unsubscribe [AT] librelist.com
```
Comment (ardumont) [inline texinfo/org-trello.org]: wow, i completely forgot about this.
it's there indeed, http://librelist.com/browser/emacs.org.trello/

**Comment by daniel-birket** [comment]
My workflow in .org is todo next work chek (review or test) and done. In this case chek items are ready for your review, although I must admit that I have not self-reviewed them as closely as usual. I was in a hurry that day.


________________________________

File: texinfo/org-trello.org [hunk 5]
```diff
@@ -610,24 +610,129 @@ remove specific existing org-trello keybindings:
...
+   This renders almost useless the checksum computation.
+   
+   Cf.  https://github.com/org-trello/org-trello-issues/271.
+   Please, do not hesitate to provide a better idea or a better implementation.
+   
+   If nil, the default, the sync to trello will be limited to what’s really changed
+   (except for the position).  So the entity’s position in trello’s board can be
+   slightly different than the one from the buffer.
+
+Orgtrello With Check On Sensible Actions
```
Comment (ardumont) [inline texinfo/org-trello.org]: I forgot this as well.
That might answer a recent question...
#391

**Comment by ardumont** [comment]
> Configuration section ready for review

Sounds good!

Thanks to you, i remember stuff i forgot ;) so thanks again.

**Comment by ardumont** [comment]
Sounds fine so far (i'm concentrating on the org buffer within commit).

**Comment by ardumont** [comment]
Any no go prior to merging this?

**Comment by daniel-birket** [comment]
I've not looked at it in a long while (other work priority) and don't recall now but I see "draft complete without How-to section" so its ok by me if you're happy with its state. Can always be fixed or improved later. Go ahead. Thank you for asking.

**Comment by ardumont** [comment]
> I've not looked at it in a long while (other work priority) and don't recall now but I see "draft complete without How-to section" so its ok by me if you're happy with its state. 

yes, i recall being fine the last time i looked ;)

> Can always be fixed or improved later. 

Indeed.

> Go ahead. Thank you for asking.

Well, thanks for your work on this ;)

**Pull request merged**